.github/workflows/deploy.yml

```diff
@@ -17,7 +17,6 @@ jobs:
           node-version: 17.x
       - name: Install and Build 🔧
         run: |
-          cd style
           npm ci --include=dev
           cp configs/config.aws.js config.js
```

Comment:
configs is currently in `src/configs`, so either this needs to change or the folder location needs to change

Reply:
Or is this run from within src?

Reply:
Nope, and this is exactly the sort of change that's impossible to test.

Fixed.